<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifiers</class>
 <widget class="QDialog" name="modifiers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/images/label.png);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:rgb(255, 255, 255);
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>id</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:rgb(255, 255, 255);
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>nbre_materiel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="type2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifierservices">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/images/modif.png)</string>
   </property>
   <property name="text">
    <string>modifier</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nbre_materiel2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nom_service2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:rgb(255, 255, 255);
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>nom_service</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_service2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:rgb(255, 255, 255);
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>type</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
